tests/neo-vm.Tests/VMJsonTestBase.cs

```diff
@@ -35,7 +35,7 @@ public void ExecuteTest(VMUT ut)
                                 {
                                     switch (run)
                                     {
-                                        case VMUTActionType.Execute: debugger.Execute(); break;
+                                        case VMUTActionType.Execute: engine.Execute(); break;
```

Comment:
`engine.Execute()` won't interrupt at break point.

Reply:
I know, but there are no test for `Execute`, and unit test don't use break points in json.
The other option is one test for `ExecutionEngine`, what do you prefer?

Reply:
I think we should use `debugger.Execute()` here, because we may use break points in the future.

We can add new test for `ExecutionEngine`.